packages/ui/components/dialog/Dialog.tsx

```diff
@@ -190,8 +192,10 @@ export function DialogFooter(props: DialogFooterProps) {
       <div
         className={classNames(
           "flex justify-end space-x-2 pb-4 pt-4 rtl:space-x-reverse",
```

Comment:
```suggestion
          "flex justify-end space-x-2 pb-4 pt-4 rtl:space-x-reverse bg-default -mx-8",
```

We can do this for now to expand the footer, like it is done in divider. Instead of hardcoding borderColor, you can add bg-default className to get desired background